src/pages/ListaColetoresScreen.jsx

```diff
@@ -0,0 +1,220 @@
+import { isCuradorOuOperador } from '../helpers/usuarios'
+
+const { confirm } = Modal
+const FormItem = Form.Item
+
+class ListaColetoresScreen extends Component {
+    columns = [
+        {
+            title: 'Nome',
+            type: 'text',
+            key: 'nome'
+        }
+    ]
+
+    constructor(props) {
+        super(props)
+        this.state = {
+            herbarios: [],
+            metadados: {},
+            loading: true,
+            pagina: 1
+        }
+
+        if (isCuradorOuOperador()) {
+            this.columns.push({
+                title: 'Ação',
+                key: 'acao'
+            })
+        }
+    }
+
+    componentDidMount() {
+        this.requisitaListaColetores({}, this.state.pagina)
+    }
+
+    gerarAcao = id => {
+        if (isCuradorOuOperador()) {
+            return (
+                <span>
+                    <Link to={`/coletores/${id}`}>
+                        <EditOutlined style={{ color: '#FFCC00' }} />
+                    </Link>
+                </span>
+            )
+        }
+    }
+
+    notificacao = (type, titulo, descricao) => {
+        notification[type]({
+            message: titulo,
+            description: descricao
+        })
+    }
+
+    formataDadosColetores = herbarios => herbarios.map(item => ({
+        key: item.id,
+        nome: item.nome,
+        acao: this.gerarAcao(item.id)
+    }))
+
+    mostraMensagemDelete(id) {
+        const self = this
+        confirm({
+            title: 'Você tem certeza que deseja excluir este herbário?',
+            content: 'Ao clicar em SIM, o herbário será excluído.',
+            okText: 'SIM',
+            okType: 'danger',
+            cancelText: 'NÃO',
+            onOk() {
+                self.requisitaExclusao(id)
+            },
+            onCancel() {
+            }
+        })
+    }
+
+    requisitaListaColetores = async (valores, pg, pageSize) => {
+        const params = {
+            pagina: pg,
+            limite: pageSize
+        }
+
+        if (valores && valores.nome) {
+            params.nome = valores.nome
+        }
+
+        try {
+            const { data } = await axios.get('/coletores', { params })
+
+            this.setState({
+                herbarios: this.formataDadosColetores(data.coletores),
+                metadados: data.metadados,
+                loading: false
+            })
+        } catch (err) {
+            this.setState({ loading: false })
+        }
+    }
+
+    handleSubmit = (err, valores) => {
+        if (!err) {
+            if (valores.nome || valores.sigla || valores.email) {
+                this.setState({
+                    valores,
+                    loading: true
+                })
+                this.requisitaListaColetores(valores, this.state.pagina)
+            } else {
+                this.notificacao('warning', 'Buscar', 'Informe ao menos um campo para realizar a busca.')
```

Comment:
Não tenho certeza se entendi, mas neste código você parece exigir que o usuário coloque pelo menos um valor nos campos de busca para fazer a busca, certo? Se for isso, muda para ficar igual as outras listagens do sistema, que quando não tem valor na busca ele não exibe erro e simplesmente faz a busca sem nenhum filtro.